perun/profile/imports.py

```diff
@@ -66,12 +165,13 @@ def import_from_string(
     )
     prof.update({"origin": minor_version.checksum})
     prof.update({"machine": get_machine_info(machine_info)})
+    prof.update({"stats": [asdict(stat) for stat in profiles.aggregate_stats(statistics.median)]}),
```

Comment:
This definitely should not be fixed to median, there is helper function that does the aggregation and supports more function and is set wrt configure option. Please, rewrite this: the `agg` parameter should preferably be completely removed and the helper function. See https://github.com/Perfexionists/perun/blob/devel/perun/utils/common/common_kit.py#L628 and https://github.com/Perfexionists/perun/blob/c2c10cc56c10e89ee56d20392c068e46e6ee7687/perun/cli.py#L697). Feel free to refactor the selection function so it is more nsync with the code (so it returns function instead of result of application of function; or add another function and maybe it could do some wrapper that would return the function based on the option from aggregate-by key).